modules/icinga2/collector.go

```diff
@@ -91,6 +106,15 @@ func Collect(c *collection.Collection) {
 		c.AddFiles(ModuleName, file)
 	}
 
+	version, err := detectIcingaVersion()
+	if err != nil {
+		c.Log.Warn("cant detect Icinga 2 version")
+	}
+
+	if version >= "2.14.0" {
```

Comment:
Maybe a comment on why this version specific for future reference. I know it's written in the issue, but people might forget.